### DISCUSSION TITLE
Nginx is not working after restart

### DISCUSSION THREAD

**Discussion by Kurra432** [learndevopsonline/learndevopsonline#1521] [Q&A]
Hi 
Nginx is not working for me after giving all private IP's , after I am restarting  the nginx  getting error 
![Nginx 2023-10-12 114155](https://github.com/learndevopsonline/learndevopsonline/assets/129031517/58226c29-b9e6-46ac-972b-fec659ac9358)

**Comment by sreedharm07** [comment]


**Reply by Kurra432** [reply]
No bro its not working
I am deleted front and again created new one it's working but I am giving the private IP's in the config file after restarting the nginx it's not working again I am getting same error
![Screenshot frontend-10-13 112829](https://github.com/learndevopsonline/learndevopsonline/assets/129031517/30170160-768e-44fa-9a7b-520b64ee6d56)

**Reply by sreedharm07** [reply]
this is duplicate ticket.
for same error solution provided in #1525
